### PR TITLE
Added all functions allowing tweet, retweet like

### LINKED ISSUE #40: implementation of the twitter service
## Describe the feature
The purpose of this feature is to implement the twitter service that will allow us to retweet, tweet or delete a tweet. All this will be done directly in conjunction with firebase Realtime.

## TO DO
creation of twitter service.
implementation of the creation and deletion of a tweet.
implementation of retweeting.
connection between firebaseRealtime & twitterService.

## Extra links (Optional)
any other features of the tweeter api desired

### PR DESCRIPTION
**Pull request by STom6**
# New feature
retweet the last tweet of a hashtag
like the last tweet of a hashtag
send a tweet
# Updated feature

# Bug fixed

### CODE REVIEW

File: Server/Services/twitterService.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
+const firebaseFunctions = require('../firebaseFunctions')
+const LoginWithTwitter = require('login-with-twitter');
+const { TwitterApi } = require('twitter-api-v2')
+
+
+//Get app token & user token, use twitter-api-v2 to send message
```
Comment (osvegn) [inline Server/Services/twitterService.js]: Create documentation following this format :
```
/**
 * @brief 
 * @param {*} req 
 * @param {*} res 
 * @param {*} appKey 
 * @param {*} appSecret 
 * @param {*} message 
 */```

**Review by 31Nathan — APPROVED** [review]
LGTM

**Review by JohanCDev — APPROVED** [review]
lgtm

**Review by osvegn — APPROVED** [review]
Look good, but app.js documentation has to be improved next time

**Review by CedricCORGE — APPROVED** [review]

**Pull request merged**